utils/securityJobSummary.go

```diff
@@ -1,553 +1,600 @@
 package utils
 
 import (
+	"errors"
 	"fmt"
 	"path/filepath"
 	"sort"
 	"strings"
 
-	"github.com/jfrog/gofrog/datastructures"
-	"github.com/jfrog/jfrog-cli-core/v2/commandsummary"
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
+
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils/commandsummary"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
 	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
 	"github.com/jfrog/jfrog-cli-security/formats"
-	"github.com/jfrog/jfrog-cli-security/formats/sarifutils"
+	"github.com/jfrog/jfrog-cli-security/resources"
 	"github.com/jfrog/jfrog-cli-security/utils/jasutils"
 	"github.com/jfrog/jfrog-cli-security/utils/severityutils"
-	"github.com/jfrog/jfrog-client-go/utils/log"
-	"github.com/jfrog/jfrog-client-go/xray/services"
-	"github.com/owenrumney/go-sarif/v2/sarif"
-	"golang.org/x/exp/maps"
 )
 
 const (
-	Build    SecuritySummarySection = "Builds"
-	Binary   SecuritySummarySection = "Artifacts"
-	Modules  SecuritySummarySection = "Modules"
-	Curation SecuritySummarySection = "Curation"
+	Build    SecuritySummarySection = "Build-info Scans"
```

Comment:
if `Modules` representes `"Source Code Scans"` - we should probably rename it to something more "source code" related - it's not clear that modules refer only to code.

Reply:
Source code are grouped under a module/Application. this value is not used now also this is just an enum